src/main/scala/com/amazon/deequ/VerificationRunBuilder.scala

```diff
@@ -82,6 +89,29 @@ class VerificationRunBuilder(val data: DataFrame) {
     this
   }
 
+  /**
+    * Use to automatically save analyzer states.
+    * Enables later aggregation e.g. when a new partition is added to the dataset.
```

Comment:
Enables aggregate computation of metrics later, e.g., when a new partition is added to the dataset.